components/lib/datatable/BodyRow.js

```diff
@@ -152,44 +164,37 @@ export const BodyRow = React.memo((props) => {
             switch (event.which) {
                 //down arrow
                 case 40:
```

Comment:
We should switch those to event.code like 'ArrowDown' etc. the old event.which is deprecated